### PR TITLE
Update 1-why-fair.md

### PR DESCRIPTION
**Pull request by MijkeJetten**
Changing the content and updating the format of the page.

### CODE REVIEW

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 1]
```diff
@@ -153,8 +155,18 @@ references:
...
+
+The added value/benefit: 
+By knowing of and adopting the FAIR principles, organisations and researchers can reach new levels of data and resource impacts, leading to numerous benefits for both the researcher community and society at large. 
+
+The impact for the research: 
+Adopting and embracing the FAIR principles is a vital step towards advancing research and addressing complex challenges in all domains. The journals, institutions, and funding bodies are now requiring more and more often that the research follows the FAIR principles ([Horizon Europe](https://horizoneuropencpportal.eu/sites/default/files/2022-09/ore-fair-data-guide.pdf) and [NIH](https://sharing.nih.gov/data-management-and-sharing-policy/data-management#:~:text=NIH%20encourages%20data%20management%20and,repurposing%20datasets%20for%20secondary%20research) requirements).
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:167]:
```suggestion
Adopting and embracing the FAIR principles is a vital step towards advancing research and addressing complex challenges in all domains. The journals, institutions, and funding bodies are often requiring that research follows the FAIR principles ([Horizon Europe](https://horizoneuropencpportal.eu/sites/default/files/2022-09/ore-fair-data-guide.pdf) and [NIH](https://sharing.nih.gov/data-management-and-sharing-policy/data-management#:~:text=NIH%20encourages%20data%20management%20and,repurposing%20datasets%20for%20secondary%20research) requirements). 
```

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 2]
```diff
@@ -43,7 +43,9 @@ content:
     - subject: "Explain what FAIR means"
       learning_outcome: 1
       activities: 
-        - "Explain what each letter in the FAIR acronym mean, and how they relate to each other"
+        - "Present to participants what each letter in the FAIR acronym means and how they relate to each other (10-15 min)"
+        - "Divide people into pairs and let them explain to each other how they are already making their data FAIR and what is one thing they can easily do to make their data FAIRer (15 min)"
+        - "To prepare for this, familiarise yourself with the FAIRisFAIR [FAIR in a nutshell](https://fairsfair.gitbook.io/fair-teaching-handbook/0lessonplans/1lessonplan) lesson plan"
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:48]:
```suggestion
        - "To prepare for this, familiarise yourself with FAIRsFAIR's [FAIR in a nutshell](https://fairsfair.gitbook.io/fair-teaching-handbook/0lessonplans/1lessonplan) lesson plan"
```

File: docs/units/FAIR-generics/topics/1-why-fair.md [hunk 3]
```diff
@@ -43,7 +43,9 @@ content:
     - subject: "Explain what FAIR means"
       learning_outcome: 1
       activities: 
-        - "Explain what each letter in the FAIR acronym mean, and how they relate to each other"
+        - "Present to participants what each letter in the FAIR acronym means and how they relate to each other (10-15 min)"
+        - "Divide people into pairs and let them explain to each other how they are already making their data FAIR and what is one thing they can easily do to make their data FAIRer (15 min)"
```
Comment (bianchini88) [inline docs/units/FAIR-generics/topics/1-why-fair.md:47]:
```suggestion
        - "Divide people into pairs and let them explain to each other how they are already making their data FAIR and what is one thing they can easily do to make their data more FAIR (15 min)"
```

**Comment by bianchini88** [comment]
@MijkeJetten @heleriinno moving this to draft so it does not get merged by mistake.

**Comment by MijkeJetten** [comment]
@heleriinno I think we should delete your branch as it is outdated. I included all its Why FAIR changes to a new branch that has been checked and is ready to be merged

**Pull request closed without merging**